app/components/widgets/Dialog.scss

```diff
@@ -59,6 +59,11 @@
       padding: 0 40px;
       margin: 0 -40px;
       max-height: 60vh;
+
+      /* Fix: non-zero bottom margin helps dynamic height rendering and prevents unnecessary scrollbars  */
```

Comment:
@ladomyriak 
The problem is only on dialogs of Hardware connect and this is changing all the dialogs (which i think logically we should avoid)
If you set the height on `HelpLinkBlock` the problem will be solved.
![image](https://user-images.githubusercontent.com/19986226/60588703-94674680-9dd2-11e9-9096-cd525cfd5922.png)

Reply:
That was my first thought too ([first commit](https://github.com/Emurgo/yoroi-frontend/pull/724/commits/82426c9e9cad7ed7e1511ecb57bac4d372525f00). The problem is the solution is very specific. We would have to solve this issue again in the future if, for example, we add a new element below the help link block, or if we remove the help link block, or if we add a component without bottom margin in another dialog, and so on and so on. I think a general solution (independent of the last element's height) is better in terms of maintainability.

Reply:
@leivaburto 
Problem was only with `HelpLinkBlock` (in modern) theme and its not something `Dialog` native problem.
That is the reason there was not any problem with other dialogs.
What is the logic behind setting only `1px` why not `2px` or so?
Try removing `HelpLinkBlock` from Hardware connect dialogs and you will see there is no problem at all.
And in future if you use `HelpLinkBlock` (in modern) at some place other than dialog you will have to fix the problem again.
cc @nicarq @nicarq

Reply:
@sarveshwar-singh Obviously there is more than one way to solve the problem. I'm sure that what you propose solves it too. The only difference is that I was trying to cover a more general scenario where other *heightless* or *marginless* elements could cause problems in `Dialog`. If you believe that `HelpLinkBlock` lack of height could cause problems at some place **other than dialog**, then I think that's material for another PR. Anyway, I think addressing both issues (default margin for elements in `Dialog` and `HelpLinkBlock`'s height) would be good as we would cover more cases 👍